tests/tasks.py

```diff
@@ -54,13 +57,14 @@ def long_task_ok():
     # Signal task has started
     conn = redis.Redis(host=REDIS_HOST, db=TEST_DB, decode_responses=True)
     conn.lpush(LONG_TASK_SIGNAL_KEY, '1')
-
+    conn.close()
```

Comment:
Didn't test, but can all of these be replaced with `with Redis() as conn`?

https://github.com/andymccurdy/redis-py/blob/master/redis/client.py#L916-L920

It makes it less likely to forget closing the connection, and it also closes explicitly in case an exception happens.